src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -308,10 +308,12 @@ public int compare(FeedbackResponseAttributes o1, FeedbackResponseAttributes o2)
                 return order;
             }
 
-            String receiverTeam1 = getTeamNameForEmail(o1.recipientEmail).isEmpty() ? getNameForEmail(o1.recipientEmail)
-                                                                                    : getTeamNameForEmail(o1.recipientEmail);
-            String receiverTeam2 = getTeamNameForEmail(o2.recipientEmail).isEmpty() ? getNameForEmail(o2.recipientEmail)
-                                                                                    : getTeamNameForEmail(o2.recipientEmail);
+            String receiverTeam1 = getTeamNameForEmail(o1.recipientEmail).isEmpty()
+                                 ? getNameForEmail(o1.recipientEmail)
+                                 : getTeamNameForEmail(o1.recipientEmail);
+            String receiverTeam2 = getTeamNameForEmail(o2.recipientEmail).isEmpty()
+                                 ? getNameForEmail(o2.recipientEmail)
+                                 : getTeamNameForEmail(o2.recipientEmail);
```

Comment:
Actually, I like the one in this document better http://yohanan.org/steve/projects/java-code-conventions/
Is the one above taken from our coding standard?